docs/visualization/plotting-overview.ipynb

```diff
@@ -232,15 +272,14 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "d['data_2d'].attrs['T'] = sc.Variable(dims=['x'], values=np.linspace(3, 50, M), unit=sc.units.K )\n",
-    "sc.plot(d['data_2d'], labels={'x': 'T'})"
+    "da1.plot(scale={'x': 'log'})"
```

Comment:
For his plot, the y range is so small that it looks like the linear plot. Should we make the temperature span a wider range, e.g. from 1 to 100?